scss/core/_variables.scss

```diff
@@ -534,27 +538,27 @@ $font-weight-bold:            700 !default;
 $font-weight-bolder:          bolder !default;
 
 $font-weight-base:            $font-weight-normal !default;
-$line-height-base:            1.5 !default;
-
-$h1-font-size:                $font-size-base * 2.5 !default;
-$h2-font-size:                $font-size-base * 2 !default;
-$h3-font-size:                $font-size-base * 1.75 !default;
-$h4-font-size:                $font-size-base * 1.5 !default;
-$h5-font-size:                $font-size-base * 1.25 !default;
-$h6-font-size:                $font-size-base !default;
-
-$h1-mobile-font-size:                $font-size-base * 2 !default;
-$h2-mobile-font-size:                $font-size-base * 1.75 !default;
-$h3-mobile-font-size:                $font-size-base * 1.5 !default;
-$h4-mobile-font-size:                $font-size-base * 1.375 !default;
-$h5-mobile-font-size:                $h5-font-size !default;
-$h6-mobile-font-size:                $h6-font-size !default;
+$line-height-base:            1.55555555555555 !default;
```

Comment:
Wait, what? Why?

Reply:
Matttttttttthhhhhssssss.  We would like our line-height base to net out at 28px like the design spec, but as Bootstrap is today I think we're required to use a unit-less line-height because some math with another unit is done with it later. 1.5555 repeating * 18px is 28px.